classes/locations.php

```diff
@@ -778,23 +788,23 @@ function ajax_save_lat_lng() {
 
 			// If we're missing a var, return false.
 			if ( empty( $_POST['sm_lat'] ) || empty( $_POST['sm_lng'] ) || empty( $_POST['sm_id'] ) ) {
-				die( __( "It doesn't look like that worked either. Please try again later.", 'SimpleMap' ) );
+				die( __( "It doesn't look like that worked either. Please try again later.", 'simplemap' ) );
```

Comment:
__[phpmd]__
* The method ajax_save_lat_lng() contains an exit expression.

<!-- Comment by [SideCI](https://sideci.com) -->